chrome/background.js

```diff
@@ -80,4 +80,51 @@ chrome.commands.onCommand.addListener(function(command) {
       }
     });
   }
+  else if (command == "move-to-new-window") {
+    chrome.permissions.request({
+      permissions: ["tabs"]
+    }, function(granted) {
+      if (granted) {
+        (() => {
+          const _currentTab = new Promise((resolve) => {
+            chrome.tabs.query({active: true, currentWindow: true}, (tabs) => {
+              resolve(tabs[0])
+            })
+          });
+
+          const _currentWindow = new Promise((resolve) => {
+            chrome.windows.getCurrent((win) => resolve(win))
+          });
+
+          (async () => {
+            const currentTab = await _currentTab
+            const currentWindow = await _currentWindow
+
+            // Let's cherrypick the options we want to clone
+            const windowOpts = {
+              tabId: currentTab.id,
+              focused: true,
+              incognito: currentWindow.incognito,
+              type: currentWindow.type
+            }
+
+            // Cant set dimensions and pos manually for some states
+            if(['minimized', 'maximized', 'fullscreen'].indexOf(currentWindow.state) !== -1)
```

Comment:
Style here does not agree with the rest of the code:
- Space after `if`.
- `{` on same line.

We can use `Array.includes` instead of `indexOf`. https://caniuse.com/#feat=array-includes

Reply:
Fixed in b506cd6.

I keep forgetting `Array.includes`. Force of habit 😏